text/networking.tex

```diff
@@ -114,8 +114,9 @@ \section{Synchronising time with SNTP}
 When the SNTP compartment receives the response and knows the time, it sets some state in a pre-shared object for detecting the time.
 
 Once the current time has been fetched, you can get the current time of day.
-\ref{lst:sntpshowtime} shows a loop that runs roughly every 50 ms and prints the time (as a UNIX epoch timestamp) if the second number of seconds has changed since last time.
-The \c{gettimeofday} function called here is from the time helpers library that mentioned earlier.
+\ref{lst:sntpshowtime} shows a loop that runs roughly every 50 ms and prints the time (as a UNIX epoch timestamp) if the number of seconds has changed since last time.
+\amanda{Okay, I am not 100% confident of this change, because my code-parsing skills are subpar.  But it does look like there's only one number of seconds each time.  If you actually meant "second number of seconds" can we change it to "new number of seconds" or something so that it's easier to parse?}
```

Comment:
```suggestion
\amanda{Okay, I am not 100\% confident of this change, because my code-parsing skills are subpar.  But it does look like there's only one number of seconds each time.  If you actually meant "second number of seconds" can we change it to "new number of seconds" or something so that it's easier to parse?}
```

% starts a comment until the end of the line, so this will break the build if I merge it because the } is commented out.

I thik the change below is fine though, if you just want to remove the comment.